nixos/modules/tasks/filesystems/btrfs.nix

```
@@ -65,6 +65,16 @@ in
         '';
       };
 
+      limit = mkOption {
+        default = null;
+        type = types.nullOr (types.strMatching "[0-9]+[KMGT]?");
```

What happens if the `[KMGT]` is omitted?

```suggestion
        type = types.nullOr (types.strMatching "[0-9]+[KMGT]");
```

---

I see, feel free to just mark as resolved then.